addons/test_mail/tests/test_mail_followers.py

```diff
@@ -147,6 +147,17 @@ def test_followers_default_partner_context(self):
             channel_ids=[self.channel_listen.id]
         )
 
+    def test_recipient_data_group_no_subtype(self):
+        """ Test that _get_recipient_data correctly fetch groups for additional pids
+        """
+        users = self.user_admin + self.user_employee + self.user_portal
+        recipient_data = self.env['mail.followers']._get_recipient_data(self.env['res.users'].browse(1), 'comment', False, pids=users.partner_id.ids)
```

Comment:
Don't hardcode an ID, and maybe use a real chatter-enabled model to have a test that makes sense.

Reply:
After a bit more tough, can't I just put `False` as it's not necessary anyway if there is as `False` subtype ?

Reply:
Would keep a message_type as this one should always be set (even if not really used in mail, only in sms with 'sms' type).